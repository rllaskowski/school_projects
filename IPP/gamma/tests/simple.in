B 5 5 3 2
m 1 0 0
m 1 1 1
m 1 3 3
m 2 1 0
m 3 2 1
m 1 0 1
m 1 2 2
g 2 1 1
b 1
b 2
b 3
f 1
f 2
f 3
p
m 3 0 4
m 3 1 4
m 3 2 4
m 3 3 4
m 3 4 4
m 3 4 3
m 3 4 2
m 3 4 1
m 3 4 0
g 3 2 2
p
