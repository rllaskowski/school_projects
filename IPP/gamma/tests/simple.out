OK 1
1
1
0
1
1
1
1
1
3
2
1
4
19
19
.....
.....
..1..
123..
12...
1
1
1
1
1
1
1
1
1
1
33333
....3
..3.3
123.3
12..3
